jina/peapods/pea.py

```diff
@@ -107,10 +107,11 @@ def __init__(self, args: Union['argparse.Namespace', Dict]):
         """ Create a new :class:`BasePea` object
 
         :param args: the arguments received from the CLI
-        :param replica_id: the id used to separate the storage of each pea, only used when ``args.separate_storage=True``
+        :param pea_id: the id used to separate the storage of each pea, only used when ``args.separate_storage=True``
         """
         super().__init__()
         self.args = args
+        self.args.pea_id = args.replica_id
```

Comment:
Is it possible to get rid of replica_id as argument as well?

Reply:
@JoanFM Updated! Took me way too long to figure out where `args` was coming from.